Classes/DDFileLogger.m

```diff
@@ -31,13 +31,15 @@
 // So we use primitive logging macros around NSLog.
 // We maintain the NS prefix on the macros to be explicit about the fact that we're using NSLog.
 
-#define LOG_LEVEL 2
+#ifndef DD_NSLOG_LEVEL
+    #define DD_NSLOG_LEVEL 2
+#endif
```

Comment:
Well, the whole point of not redefining it if it's already defined is that it could be defined earlier, like let's say in Xcode build settings for a specific configuration (DEBUG). In this case, the naming is important to not be confusing.